eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -289,10 +289,10 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
             firstPerCommitmentPoint = keyManager.commitmentPoint(localParams.channelKeyPath, 0))
           val remoteParams = RemoteParams(
             nodeId = remoteNodeId,
-            dustLimitSatoshis = open.dustLimitSatoshis,
+            dustLimitSatoshis = open.dustLimitSatoshis.toLong,
             maxHtlcValueInFlightMsat = open.maxHtlcValueInFlightMsat,
-            channelReserveSatoshis = open.channelReserveSatoshis, // remote requires local to keep this much satoshis as direct payment
-            htlcMinimumMsat = open.htlcMinimumMsat,
+            channelReserveSatoshis = open.channelReserveSatoshis.toLong, // remote requires local to keep this much satoshis as direct payment
```

Comment:
this is supposed to be temporary, right ?

Reply:
Absolutely! It's temporary until `LocalParams` and `RemoteParams` are migrated to the typed amounts.